models/general-models.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs.promises')
+
+exports.fetchEndpoints = () => {
```

Comment:
The model is unnecessary in this instance as no db query is happening. You can get rid of it if you want, and have your readFile in the controller.